RELEASING.md

```diff
@@ -0,0 +1,19 @@
+# RELEASING
+
+This library is available via Swift Package Manager and CocoaPods.  
+Once you have [configured your setup](#setup), execute the following to
+make a release:
+
+ - Tag the commit as a release: `git tag x.x.x`
```

Comment:
Very nice.

Some questions: Isn't it nice to have the changed `.podspec` also be part of the tagged commit?

Also: Couldn't this be done automatically by github actions on the with `on: tag`?

Reply:
Yes, I'll reorder the commands 💯 

I'll hold back on the github actions part, to be on the safe side with permissions and such